boards/mediatek/mt8195_adsp/Kconfig.mt8195_adsp

```diff
@@ -2,6 +2,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_MT8195_ADSP
-	select SOC_MT8195_ADSP
 	help
 	  Board with Mediatek MT8195 Audio DSP
```

Comment:
??? this cannot work

Reply:
Clarify?  I mean, it builds OK for me.  The 8195 code is in-tree already and predates hwmv2.  My guess is this was left in during the transition and I'm just removing it for hygiene?